### PR TITLE
Add Coding Ninjas icon

### LINKED ISSUE #4967: Brand Name: Coding Ninjas
**Brand Name:** Coding Ninjas
**Website:** https://www.codingninjas.com
**Alexa rank:** 15054
**Official resources for icon and colour:** https://www.codingninjas.com/assets-landing/images/CNLOGO.svg

### PR DESCRIPTION
**Pull request by service-paradis**
image

**Issue:** Close #4967
**Alexa rank:** [~15k](https://www.alexa.com/siteinfo/codingninjas.com)
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

I gave this one a try!

- I took `Coding Ninjas Logo 1` from their [media kit](https://www.codingninjas.com/press-release)
- I used Illustrator to vectorize the logo
- I used Inkscape to clean, adjust and scale
- I used https://jakearchibald.github.io/svgomg/ to optimize
- I took the color from their "full" logo in the header of their [website](https://www.codingninjas.com/)
  - It can be discussed since they use a lot of different orange shades

### CODE REVIEW

**Review by PeterShaggyNoble — APPROVED** [review]
SVG looks spot on to me, @service-paradis :+1: Nice work. And I'd agree with your choice of colour being our best option.

As this is a manual vectorisation, we'll need on of the other @simple-icons/maintainers to throw an eye over it before merging.

**Comment by fbernhart** [comment]
LGTM 👍🏼 Great work @service-paradis 🏆

**Pull request merged**
